### PR TITLE
Try GitHub pull request trigger actions

### PR DESCRIPTION
**Pull request by christianhueserhzdr**
Changed GitHub action events which trigger pipelines in LabHub TOML file

The actions which trigger pipelines are 
* synchronize
* opened
* reopened
* closed

Please check if that is correct.

All possible actions are listed here:
https://developer.github.com/v3/activity/events/types/#pullrequestevent

### CODE REVIEW

File: src/actions/actions.rs
Comment (tobiashuste) [inline src/actions/actions.rs]: One question: in case somebody defines non-default configuration options in his `LabHub.toml`, would this still work or do all possible options need to be mentioned here?

File: LabHub.toml
Comment (tobiashuste) [inline LabHub.toml:46]: To me, this list looks totally fine. When we submit the PR to the original repo, it might be possible that they request us to add the complete list here in order to not cause a different behavior than before.

But let's wait for their review and see what they say.

**Review by tobiashuste — COMMENTED** [review]
We should squash the commits before merging to master. Thanks, for working on this :)

File: src/actions/actions.rs
Comment (christianhueserhzdr) [reply src/actions/actions.rs]: Yes, it will still work. The TOML file may contain lots of entries, but only those which are actual actions will be taken into account.

**Review by tobiashuste — APPROVED** [review]

File: LabHub.toml
Comment (christianhueserhzdr) [reply LabHub.toml:46]: Alright, you are right. We need to put everything in here for the original repo.

Comment (tobiashuste) [reply LabHub.toml:46]: But let's just wait for their reaction first. To me, this new behavior makes more sense.

File: src/actions/actions.rs
Comment (christianhueserhzdr) [reply src/actions/actions.rs]: Ok, I see what you mean. It might be reasonable to add the complete list since this is no config file.

**Comment by tobiashuste** [comment]
@christianhueserhzdr I rebased to the current master. Can you squash and merge, please?

**Pull request merged**
